### DISCUSSION TITLE
I've created a rofi builder docker image

### DISCUSSION THREAD

**Discussion by fvigotti** [davatorium/rofi#1293] [Show and tell]
I needed a binary to replace the one available on the os which is buggy.. I haven't been able to find an updated static binary , and the compilation instructions seemed a bit incomplete ( I have had to add some libraries ) /partial ( maybe just my os is wrong )  + a better solution is to build in a temp docker container so you don't add too many moving parts to your os..
so here it is.. for those who may need it .. 

https://gist.github.com/fvigotti/7369344aca18056687adb1645695a203

if you have a better/easier solution let me know,
have a nice day :)

**Comment by alabnt** [comment]
This seems to be for Rofi 1.6.1?
